test/extended/prometheus/prometheus.go

```diff
@@ -71,6 +71,14 @@ var _ = g.Describe("[sig-instrumentation][Late] OpenShift alerting rules", func(
 	var alertingRules map[string][]promv1.AlertingRule
 	oc := exutil.NewCLIWithoutNamespace("prometheus")
 
+	g.BeforeEach(func() {
```

Comment:
please enumerate which tests are being skipped and why an apigroup detection or capability detection (do I have a prometheus route) isn't more appropriate.